### PR TITLE
Bootstrap organisers and avatar shared partials

### PR DESCRIPTION
**Pull request by KimberleyCook**
Bootstrap organisers and avatar shared partials.

### CODE REVIEW

File: app/views/members/_avatar_listing.html.haml [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
-      %br
-      %br
-      = member.full_name
-      - if show_info
-        = member.mobile
+- member.each do |member|
+  .col-sm-4.col-md-3.mb-4
+    =link_to twitter_url_for(member.twitter), class: 'user-link' do
+      =image_tag(member.avatar(56), class: 'rounded-circle', title: member.full_name, alt: member.full_name)
+    %p.mt-3.mb-1= member.full_name
```
Comment (KimberleyCook) [inline app/views/members/_avatar_listing.html.haml:5]: I removed the telephone number, as I'm not sure we should be sharing this with everyone on the workshop page publically, only in the admin area for the page once someone has RSVP'd.

File: db/schema.rb [hunk 2]
```diff
@@ -129,8 +129,8 @@
     t.string   "name"
     t.string   "surname"
     t.string   "email"
-    t.boolean  "mailing_list_consent",        default: false
-    t.string   "token",                                       null: false
+    t.boolean  "mailing_list_consent", default: false
+    t.string   "token", null: false
```
Comment (KimberleyCook) [inline db/schema.rb:133]: This was failing the rake tests again, so had to update.

Once this PR or [this one](https://github.com/codebar/planner/pull/1435) are merged in, I will no longer need to update this.

File: app/views/members/_avatar_listing.html.haml (hunk 1 above)
Comment (gnclmorais) [reply app/views/members/_avatar_listing.html.haml:5]: Sounds good 👍

Comment (despo) [reply app/views/members/_avatar_listing.html.haml:5]: @KimberleyCook could we apply this separately in a new PR and not as part of the boostrap changes though.

**Review by matyikriszta — APPROVED** [review]
I think this looks good now 👍

**Pull request merged**